### ISSUE TITLE
Question: 

### ISSUE THREAD

**Issue by BoyfriendFNF** [Verwex/Funkin-Mic-d-Up-SC#4]
## What is your question?
An possible android port? There's already a Friday Night Funkin android port [right here. ](https://github.com/luckydog7/Funkin-android) and you just need to follow the build instructions. Please?

**Comment by Verwex** [comment]
This would mean a lot of event changes so don't expect a port. It's anot as simple as building it for Android, I made it desktop exclusive. Besides, I don't think FNF belongs on mobile just because of how weird it'd be to play on THIS port.

TLDR: This port sucks to play and it's desktop exclusive for a reason so no.

**Comment by BoyfriendFNF** [comment]
> This would mean a lot of event changes so don't expect a port. It's not as simple as building it for Android, I made it desktop exclusive and took advantage of it. Besides, I don't think THIS port is fun to play either.
> 
> TLDR: This port sucks to play and the mod's desktop exclusive for a reason so no.

oh ok

**Comment by zacksgamerz** [comment]
Damn.
K then.

**Comment by KlavierGayming** [comment]
man.
